source/codegen/templates/service_helpers.mako

```diff
@@ -491,8 +491,15 @@ ${initialize_standard_input_param(function_name, parameter)}
 % else:
 <%
   size_field_name = size_sources[-1]
+  size_from_param_name = parameter["determine_size_from"][0] if isinstance(parameter["determine_size_from"], list) else parameter["determine_size_from"]
+  size_field_param = common_helpers.get_param_with_name(parameters, size_from_param_name)
```

Comment:
I wouldn't call this the "size_field_param". More like the `linked_array_param`?

Reply:
I had named this based on `size_field_name` above. I have renamed it to `linked_array_param` now.